packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructService.ts

```diff
@@ -292,7 +294,7 @@ class IndigoService implements StructService {
         data: commandData,
       };
 
-      this.EE.removeAllListeners(WorkerEvent.Convert);
```

Comment:
checked with this change's issue. works ok https://github.com/epam/ketcher/issues/3024